test/jest/unit/snyk-code/snyk-code-test.spec.ts

```
@@ -929,6 +934,33 @@ describe('Test snyk code', () => {
       'Missing configuration for Snyk Code Local Engine. Refer to our docs on https://docs.snyk.io/products/snyk-code/deployment-options/snyk-code-local-engine/cli-and-ide to learn more',
     );
   });
+
+  it('Local code engine - makes GET /status to get SCLE version', async () => {
+    const sastSettings = {
+      sastEnabled: true,
+      localCodeEngine: {
+        url: 'http://foo.bar',
+        allowCloudUpload: true,
+        enabled: true,
+      },
+    };
+    await getCodeTestResults(
+      '.',
+      {
+        path: '',
+        code: true,
+      },
```

Q: does this test past without the debug option being set?

---

@nick-watson nope, that wasn't working without the debug option. I fixed it.

---

@metju90 👍 do we also need a test cases for when it's not passed as well as if an error is thrown?

---

I think it's a bit of an overkill to test every aspect of a logging feature?

---

I think we should at least have a test checking/documenting that any errors thrown don't cause the analysis to fail early

---

Done